infra/terraform/test_fixtures/terraform-google-iam.tf

```diff
@@ -0,0 +1,80 @@
+locals {
+  // Roles required by CI service accounts to run the IAM
+  // integration tests.
+  iam_required_roles = [
+    "roles/resourcemanager.organizationAdmin",
+    "roles/compute.xpnAdmin",
```

Comment:
This permission isn't mentioned in the [project README](https://github.com/terraform-google-modules/terraform-google-iam). Does this need to be updated?

Reply:
Technically we can say it's mentioned not explicitly, compute.xpnAdmin contains those permissions:

```
Project custom: Add compute.subnetworks.getIamPolicy	and compute.subnetworks..setIamPolicy permissions.
```